.changeset/poor-teachers-allow.md

```diff
@@ -0,0 +1,6 @@
+---
+"@khanacademy/perseus": minor
+"@khanacademy/perseus-editor": minor
+---
+
+[Interactive Graph Editor] Implement "Add locked circle" UI
```

Comment:
I like how you preface the changeset like this, I'm going to start copying you

Reply:
Thanks to @jeremywiebe for the idea!